prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -529,7 +610,13 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct fi_cq_data_entry *
 	peer = rxr_ep_get_peer(ep, pkt_entry->addr);
 	if (!peer->is_local)
 		rxr_ep_dec_tx_pending(ep, peer, 0);
-	rxr_pkt_entry_release_tx(ep, pkt_entry);
+
+	/*
+	 * If ptk_entry->state == RXR_PKT_ENTRY_COPY_BY_READ, the
+	 * handle_data_copied function will release pkt_entry
+	 */
+	if (pkt_entry->state == RXR_PKT_ENTRY_IN_USE)
```

Comment:
What about `RXR_PKT_ENTRY_RNR_RETRANSMIT`? Shouldn't this check just be `!= RXR_PKT_ENTRY_COPY_BY_READ` instead?